cmd/govim/testdata/scenario_default/goproxytest_usage.txt

```diff
@@ -1,20 +1,19 @@
 # A simple test that verifies the setup of goproxytest
 
 vim ex 'e main.go'
-errlogmatch 'PublishDiagnostics callback: &protocol.PublishDiagnosticsParams{\n\S+:\s+URI:\s+"file://'$WORK/main.go
 vim ex 'w'
```

Comment:
Because we write `main.go` here, which itself calls `GOVIMGoImports`, things correctly block on the `gopls` side to make a wait/watch here redundant.